stable/overwrite/README.md

```diff
@@ -1,62 +1,64 @@
 [![Docs](https://img.shields.io/badge/docs-current-brightgreen.svg)](https://godoc.org/gopkg.in/dedis/kyber.v1)
 [![Build Status](https://travis-ci.org/dedis/kyber.svg?branch=v1)](https://travis-ci.org/dedis/kyber)
 
-DeDiS Advanced Crypto Library for Go
+DEDIS Advanced Crypto Library for Go
 ====================================
 
 This package provides a toolbox of advanced cryptographic primitives for Go,
-targeting applications like [Dissent](http://dedis.cs.yale.edu/dissent/)
+targeting applications like [Cothority](https://github.com/dedis/cothority)
 that need more than straightforward signing and encryption.
 Please see the
-[GoDoc documentation for this package](http://godoc.org/gopkg.in/dedis/kyber.v1)
+[Godoc documentation for this package](http://godoc.org/gopkg.in/dedis/kyber.v1)
 for details on the library's purpose and API functionality.
 
-Versioning - V1
----------------
+Versioning - Development
```

Comment:
Is this the dedis/kyber/README.md? I'd make it more clear that this branch is the 'stable' `v1`-branch, as this will be put in `gopkg.in/dedis/kyber`.